app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -170,9 +180,18 @@ public Study getStudy(String identifier, boolean includeDeleted) {
             study = studyDao.getStudy(identifier);
             cacheProvider.setStudy(study);
         }
-
-        if (study != null && !study.isActive() && !includeDeleted) {
-            throw new EntityNotFoundException(Study.class, "Study not found.");
+        if (study != null) {
+            // If it it exists and has been deactivated, and this call is not supposed to retrieve deactivated
+            // studies, treat it as if it doesn't exist.
+            if (!study.isActive() && !includeDeleted) {
+                throw new EntityNotFoundException(Study.class, "Study not found.");
+            }
+            // Because this template does not currently exist in studies, add the default if it is null.
+            if (study.getEmailSignInTemplate() == null) {
+                EmailTemplate template = new EmailTemplate(defaultEmailSignInTemplateSubject,
+                        defaultEmailSignInTemplate, MimeType.HTML);
+                study.setEmailSignInTemplate(template);
+            }
```

Comment:
Is there any way to test this? Right now, all the existing tests go through create() or update(), which means this code path is not being hit in any of our tests.